src/app/ngrx/effects/routing.effects.ts

```diff
@@ -156,6 +168,11 @@ export class RoutingEffects {
               private actions$: Actions) {
     this.store.select(selectRouter).subscribe(routerState => this.routerState = routerState);
     this.store.select(selectPodcasts).subscribe(podcasts => this.podcasts = podcasts);
+    this.router.events.filter(event => event instanceof RoutesRecognized).subscribe((event: RoutesRecognized) => {
```

Comment:
This is the big change. Catch router events to `'/'` and route back to the previous state. This does not give us a full
 cycle of router events on the route back to previous state though, hence the weirdness with customeRouterNavigation$ above.

Reply:
By "previous state" do you mean it acts like a Back button?

Reply:
Sort of. More like it backs out of actually routing to '/'. The app doesn't seem to actually change. The route appears to stay the same, and there is no subsequent router event because it matches the existing actual route.